src/app/[lng]/learn/page.tsx

```diff
@@ -1,6 +1,14 @@
 import { getArticles } from '@/app/lib/getArticles';
 import { getFeaturedArticle } from '@/app/lib/getFeaturedArticle';
 import LearnPage from '@/app/ui/learn/LearnPage';
+import { Metadata } from 'next';
+
+export async function generateMetadata(): Promise<Metadata> {
+  return {
+    title: 'Jumper Learn',
+    description: 'Jumper Learn is the blog of Jumper Exchange.',
```

Comment:
May be a first shot, but I am not convinced this is a good description. Probably this is not important, but seeing this does not get me any excited.